.github/workflows/security_trivy.yml

```diff
@@ -31,13 +31,13 @@ jobs:
         format: 'sarif'
         output: 'trivy-results.sarif'
         exit-code: 1
-        limit-severities-for-sarif: true
```

Comment:
note that this will cause the trivy tests to fail and thus I can test the slack notification 😁